clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -221,6 +219,26 @@ func (p *profileMeasurement) shouldExposeApiServerDebugEndpoint() bool {
 	return p.config.componentName == "kube-apiserver"
 }
 
+func (p *profileMeasurement) getProfileCommand() (string, error) {
+	profilePort, err := getPortForComponent(p.config.componentName)
+	if err != nil {
+		return "", fmt.Errorf("get profile command failed finding component port: %v", err)
+	}
+	profileProtocol := getProtocolForComponent(p.config.componentName)
+
+	//TODO: use flags for etcdCert and etcdKey
+	var command string
+	if p.config.componentName == "etcd" {
+		etcdCert := "/etc/srv/kubernetes/pki/etcd-apiserver-server.crt"
```

Comment:
The `Execute` method has the MeasurementConfig argument. You can access the ClusterConfig from there (via ClusterLoaderConfig, IIRC). Then you can either pipe the values you need here, or add them to the profileMeasurement struct.

Reply:
@skurtzemann - `start()` method already has access to the config, and that is calling `gatherProfile` - let's just pipe them here

Reply:
Ok, thanks for your help. I submit the updated PR with the config